ksql-execution/src/main/java/io/confluent/ksql/execution/codegen/SearchedCaseFunction.java

```diff
@@ -13,7 +13,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package io.confluent.ksql.function.udf.caseexpression;
+package io.confluent.ksql.execution.codegen;
```

Comment:
Can we put this into a sub package for inbuilt functionality? e.g. `functions` or some such?